pallets/cash/src/lib.rs

```diff
@@ -191,6 +193,12 @@ decl_storage! {
 
         /// Validator spread due to miner of last block.
         LastMinerSharePrincipal get(fn last_miner_share_principal): CashPrincipalAmount;
+
+        /// The timestamp of the previous block or defaults to timestamp at genesis.
+        LastBlockTimestamp get(fn last_block_timestamp): Timestamp;
+
+        /// The cash index of the previous yield accrual point or defaults to initial cash index.
+        LastYieldCashIndex get(fn last_yield_cash_index): CashIndex;
```

Comment:
I think using Default::default() = cash index of 1 for now makes sense and we don't need `config()` or `add_extra_genisis` for this right now.